aicoe/sesheta/utils.py

```diff
@@ -38,6 +38,55 @@
 THOTH_DEVOPS_SPACE = os.getenv("SESHETA_THOTH_DEVOPS_SPACE", None)
 AIOPS_DEVOPS_SPACE = os.getenv("SESHETA_AIOPS_DEVOPS_SPACE", None)
 
+GITHUB_REALNAME_MAP = {
+    "elmiko": "Michael McCune",
+    "sub-mod": "Subin Modeel",
+    "xtuchyna": "Dominik Tuchyna",
+    "kpostoffice": "Kevin Postlethwait",
+    "harshad16": "Harshad Reddy Nalla",
+    "goern": "Christoph Goern",
+    "shruthi-raghuraman": "Shruthi Raghuraman",
+    "anishasthana": "Anish Asthana",
+    "fridex": "Frido Pokorny",
+    "llunved": "Daniel Riek",
+    "pacospace": "Francesco Murdaca",
+    "4n4nd": "Anand Sanmukhani",
+    "bissenbay": "Bissenbay Dauletbayev",
+    "shuels": "Steven Huels",
+    "ddehueck": "Devin de Hueck",
+    "dfeddema": "Diane Feddema",
+    "durandom": "Marcel Hild",
+    "cermakm": "Marek Cermak",
+    "hemajv": "Hema Veeradhi",
+    "zmhassan": "Zak Hassan",
+    "humairak": "Humair Khan",
+    "sesheta": "Thoth Bot",
+}
+
+REALNAME_HANGOUTS_MAP = {
```

Comment:
I don't if we should add **user_ids** in the code, may using the api logic to get the user_id as each query sounds compute, as we don't emphasis on the compute time. 
May be we should do a lookup of member from channel.
Logic in older code: 
```
response = chat.spaces().members().list(parent=<channel_space>)
result = response.execute()
for member in result["memberships"]:
    if member["member"].get("displayName") == <user>:
        return member["member"].get("name")
```

Reply:
:) feel free to approve this and send a patch :package: